docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,191 @@
+    
+    mv -Force tools.conf 'C:\ProgramData\VMware\VMware Tools\tools.conf'
+```
+
+## 3. Set up SSH
+
+Install and configure the OpenSSH Server on the Windows virtual machine with automatic startup and key based-authentication.
+See [Microsoft documentation](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
+to install using PowerShell.
+
+The provided PowerShell script [install-openssh.ps1](vsphere_ci/scripts/install-openssh.ps1) aims to programmatically
+install and configure the OpenSSH Server with the public key file from section [Add the Administrator 
+Public Key](#add-the-administrator-public-key).
+
+Ensure the SSH service is running and that it will start on boot by running the following PowerShell command:
+
+```powershell
+    Get-Service -Name "ssh*" | Select Name, Status, StartType 
+```
+
+where, both installed services *ssh-agent* and *sshd* must have status *Running* and start type *Automatic*.
+
+Ensure the OpenSSH Server installation successfully created an inbound firewall rule enabling access to the above 
+service (sshd) by running the following PowerShell command:
+
+```powershell
+    Get-NetFirewallRule -DisplayName "*ssh*"
+```
+
+In case no firewall rule exist, you must create it by running the following PowerShell command:
+
+```powershell
+    New-NetFirewallRule -DisplayName 'OpenSSH Server (sshd)' -LocalPort 22 -Enabled True -Direction Inbound -Protocol TCP -Action Allow 
+```
+
+### Add the Administrator Public Key
+
+This section provides an overview of how to use key-based authentication with OpenSSH server, similar to Linux 
+environments commonly setup of public-key/private-key pairs to drive authentication without passwords. Key pairs refer 
+to the public and private key files that are used by certain authentication protocols.
+
+**Important**: You need to have OpenSSH Server installed first. Please see [Set up SSH](#3-set-up-ssh).
+
+#### Private key generation
+
+To use key-based authentication, you first need to generate public/private key pairs. 
+
+As the private key file let's reuse the exiting [the secret](../README.md#create-a-private-key-secret) created while 
+installing WMCO, along with the corresponding public key file.
+
+#### Deploying the public key
+
+The contents of the public key file needs to be placed on the Windows VM as a text file called 
+`administrators_authorized_keys` in the *C:\ProgramData\ssh\\* directory to allow administration access via SSH, as 
+describe by the [Microsoft documentation.](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#administrative-user) 
+
+Alternatively, a PowerShell script [install-openssh.ps1](vsphere_ci/scripts/install-openssh.ps1) is provided to 
+automate the process of installing and configuring the OpenSSH Server with the public key file. For example, you can 
+the following command:
+
+```
+    ./install-openssh.ps1 <path/to/public_key_file>
+```
```

Comment:
You have already mentioned this in step 3. So you can move the command up to step 3 and remove it from here and just mention this step is not needed if the script was used in step 3.

Reply:
Good catch, thank you!